src/components/ImageScroller.js

```diff
@@ -103,6 +126,19 @@ export default function ImageScroller(props) {
     });
   }
 
+  function handleDataChange(img) {
+    if (img) {
```

Comment:
Doing the selection and rotation at the same time doesn't work. Only the rotation is applied.
Selection and rotation should be two separate actions.